pkg/cachedinstances/factory.go

```diff
@@ -1,85 +0,0 @@
-package cachedinstances
```

Comment:
This file is refactored into the one above ^ (`pkg/cachedinstances/cachedinstances.go`)